admin/lib/tasks/tailwind.rake

```diff
@@ -0,0 +1,39 @@
+namespace :solidus_admin do
+  namespace :tailwindcss do
```

Comment:
optional nit: thoughts on using "tailswindcss" also for the filename?

Reply:
Makes total sense. Changed!